### ISSUE TITLE
Should we include the validation for names/forenames?

### ISSUE THREAD

**Issue by bravegrape** [alphagov/dcs-pilot-docs#92]
Suggestion from @bjgill : Would it be useful to include the exact regex we use to validate surnames/forenames.

This issue is to log the suggestion. We aim to research this as part of usability testing.

**Comment by pauldougan** [comment]
not sure that we need to go as far as including a regex, an explanation in plain english might be better given that regexes are not notoriously legible :)

**Comment by mrwilson** [comment]
We do this now, so I'm going to close this. :)

https://dcs-pilot-docs.cloudapps.digital/api-reference/#surname
https://dcs-pilot-docs.cloudapps.digital/api-reference/#forenames
